bin/gw_summary

```diff
@@ -457,6 +458,17 @@ os.chdir(opts.output_dir)
 plotdir = os.path.join(path, 'plots')
 mkdir(plotdir)
 
+# add file containing a list of directories if required
+if (mode.get_mode().is_calendar() and
+        config.has_option("calendar", "highlight-available")):
+    # directory to list
+    dirpath = os.path.split(path)[0]
+    datedirs = sorted([thisdir for thisdir in os.listdir(dirpath)
+                       if os.path.isdir(os.path.join(dirpath, thisdir))])
+    dirfile = os.path.join(dirpath, 'list-dirs.txt')
```

Comment:
@mattpitkin, this use of a txt file to pass dates seems a little clunky, can you just pass the `datedirs` object directly?

Reply:
I think it has to be done this way, so that all summaries that you've created see the same set of dates. Otherwise, when you go to summaries on different dates you won't get all the dates highlighted each time, you'll only get the ones that were present when the summary was created.

Reply:
Aha, that's clever, your method makes a lot more sense to me now, thanks! I'll take a look at your change to pass this file as part of the configuration.